osu.Game.Rulesets.Osu/Mods/OsuModHidden.cs

```diff
@@ -34,7 +36,30 @@ public override void ApplyToDrawableHitObjects(IEnumerable<DrawableHitObject> dr
                     adjustFadeIn(h);
             }
 
-            base.ApplyToDrawableHitObjects(drawables);
+            int skipCount = 0;
+
+            if (IncreaseFirstObjectVisibility.Value)
+            {
+                ++skipCount;
+
+                int spinnerCount = checkForSpinners(drawables);
+
+                if (spinnerCount > 0)
+                {
+                    IEnumerable<DrawableHitObject> startingSpinners = drawables.Take(spinnerCount);
+
+                    foreach (var d in startingSpinners)
+                        d.ApplyCustomUpdateState += ApplyHiddenState;
+
+                    skipCount += spinnerCount;
+                }
+
+                DrawableHitObject firstHitObject = drawables.First();
```

Comment:
I have forgotten to skip the spinners, so the first drawable I was taking is actually a spinner. Weird, *was* working as intended before